main.go

```diff
@@ -35,17 +38,23 @@ func main() {
 
 	client := teamcity.New(*hostname, *username, *password)
 
-	b, err := client.QueueBuild(*configId, "master", nil)
+	properties := make(map[string]string)
+	for _, pair := range *jobParams {
+		keyValue := strings.Split(pair, "=")
```

Comment:
maybe `SplitN(pair, "=", 2)` or some assert 
```
if len(keyValue) != 2 {
    log.Panic(...)
}
```

but it's small detail

Reply:
Added, merging.